packages/account/src/account.ts

```diff
@@ -433,43 +439,14 @@ export class Account extends AbstractAccount {
     assetId?: BytesLike,
     txParams: TxParamsType = {}
   ): Promise<TransactionResponse> {
-    if (bn(amount).lte(0)) {
-      throw new FuelError(
-        ErrorCode.INVALID_TRANSFER_AMOUNT,
-        'Transfer amount must be a positive number.'
-      );
-    }
-
-    const contractAddress = Address.fromAddressOrString(contractId);
-    const assetIdToTransfer = assetId ?? this.provider.getBaseAssetId();
-    const { script, scriptData } = await assembleTransferToContractScript({
-      hexlifiedContractId: contractAddress.toB256(),
-      amountToTransfer: bn(amount),
-      assetId: assetIdToTransfer,
-    });
-
-    let request = new ScriptTransactionRequest({
-      ...txParams,
-      script,
-      scriptData,
-    });
-
-    request.addContractInputAndOutput(contractAddress);
-
-    const txCost = await this.getTransactionCost(request, {
-      quantities: [{ amount: bn(amount), assetId: String(assetIdToTransfer) }],
-    });
-
-    request = this.validateGasLimitAndMaxFee({
-      transactionRequest: request,
-      gasUsed: txCost.gasUsed,
-      maxFee: txCost.maxFee,
-      txParams,
-    });
-
-    await this.fund(request, txCost);
+    return this.peformTransferToContracts([{ amount, assetId, contractId }], txParams);
+  }
 
-    return this.sendTransaction(request);
+  async batchTransferToContracts(
+    contractTransferParams: ContractTransferParams[],
+    txParams: TxParamsType = {}
+  ): Promise<TransactionResponse> {
+    return this.peformTransferToContracts(contractTransferParams, txParams);
```

Comment:
Why not delete the `peformTransferToContracts()` method and move its contents inside `batchTransferToContracts()`?

Reply:
Makes sense: https://github.com/FuelLabs/fuels-ts/pull/3335/commits/f75848827061acd2b80a8bb6e5f34822f648412b